src/main/java/in/twizmwaz/cardinal/command/TeamCommands.java

```diff
@@ -0,0 +1,142 @@
+package in.twizmwaz.cardinal.command;
+
+import in.twizmwaz.cardinal.GameHandler;
+import in.twizmwaz.cardinal.chat.ChatConstant;
+import in.twizmwaz.cardinal.chat.LocalizedChatMessage;
+import in.twizmwaz.cardinal.chat.UnlocalizedChatMessage;
+import in.twizmwaz.cardinal.event.TeamNameChangeEvent;
+import in.twizmwaz.cardinal.module.modules.team.TeamModule;
+import in.twizmwaz.cardinal.util.ChatUtils;
+import in.twizmwaz.cardinal.util.TeamUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import com.sk89q.minecraft.util.commands.Command;
+import com.sk89q.minecraft.util.commands.CommandContext;
+import com.sk89q.minecraft.util.commands.CommandException;
+import com.sk89q.minecraft.util.commands.CommandPermissions;
+import com.sk89q.minecraft.util.commands.CommandUsageException;
+import com.sk89q.minecraft.util.commands.NestedCommand;
+
+public class TeamCommands {
+
+    public static class TeamParentCommand {
+        @Command(
+                aliases = { "team" },
+                desc = "Manage the teams in the match.",
+                min = 1,
```

Comment:
Why'd you have a min and max here? You should be able to do a min and max specific to each nested command, right?